src/display/editor/tools.js

```diff
@@ -1308,6 +1373,49 @@ class AnnotationEditorUIManager {
     });
   }
 
+  translateSelectedEditors(x, y) {
+    this.commitOrRemove();
+    if (!this.hasSelection) {
+      return;
+    }
+
+    this.#translation[0] += x;
+    this.#translation[1] += y;
+    const [totalX, totalY] = this.#translation;
+    const editors = [...this.#selectedEditors];
```

Comment:
Given that this is accessed in a `setTimeout` below, is there perhaps a risk that an editor could actually have been e.g. removed *before* the timeout triggers?
That situation is obviously not easy to trigger, but does the code still work correctly if that were to happen or would we be adding "unneeded" undo commands in the removed-case?